### PR TITLE
Store channelSid serviceSid

### PR DESCRIPTION
**Pull request by murilovmachado**
Jira: https://bugs.benetech.org/browse/CHI-712
Backend PR: https://github.com/techmatters/hrm/pull/118

This PR stores `channelSid` and `serviceSid` when saving a contact
For voice calls or offline contacts, those values are not saved.

### CODE REVIEW

File: plugin-hrm-form/src/HrmFormPlugin.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ export const getConfig = () => {
   const hrmBaseUrl = `${manager.serviceConfiguration.attributes.hrm_base_url}/${manager.serviceConfiguration.attributes.hrm_api_version}/accounts/${manager.workerClient.accountSid}`;
   const serverlessBaseUrl = manager.serviceConfiguration.attributes.serverless_base_url;
   const logoUrl = manager.serviceConfiguration.attributes.logo_url;
+  const serviceSid = manager.serviceConfiguration.chat_service_instance_sid;
```
Comment (nickhurlburt) [inline plugin-hrm-form/src/HrmFormPlugin.js:35]: Should this be `chatServiceSid`?  Just a thought.  Up to you.

File: plugin-hrm-form/src/services/ContactService.ts [hunk 2]
```diff
@@ -195,6 +201,14 @@ export async function saveToHrm(task, form, workerSid, workerHelpline, uniqueIde
 
   const helplineToSend = (isOfflineContactTask(task) && form.contactlessTask?.helpline) || workerHelpline;
 
+  let channelSid;
+  let serviceSid;
+
+  if (isTwilioTask(task) && TaskHelper.isChatBasedTask(task)) {
+    ({ channelSid } = task.attributes);
+    ({ serviceSid } = getConfig());
```
Comment (nickhurlburt) [inline plugin-hrm-form/src/services/ContactService.ts:209]: Are the outer parentheses necessary?  Just curious.

**Review by nickhurlburt — APPROVED** [review]
Awesome!

File: plugin-hrm-form/src/HrmFormPlugin.js (hunk 1 above)
Comment (murilovmachado) [reply plugin-hrm-form/src/HrmFormPlugin.js:35]: Good call!

**Review by GPaoloni — APPROVED** [review]
Looks good to me!

**Pull request merged**